ci/scripts/python_wheel_manylinux_build.sh

```diff
@@ -160,6 +160,26 @@ export CMAKE_PREFIX_PATH=/tmp/arrow-dist
 pushd /arrow/python
 python setup.py bdist_wheel
 
+echo "=== Strip symbols from wheel ==="
+mkdir dist/temp-fix-wheel
```

Comment:
Let's avoid erroring out if the directory already exists?
```suggestion
mkdir -p dist/temp-fix-wheel
```